ci/cdn-broker-certs.sh

```diff
@@ -4,11 +4,12 @@ set -euxo pipefail
 
 ## Calculate the days to expiration from now
 function days_to_cert_expiration {
-  local target=$1;
-  local date_today=$(date +%s)
+  local client=$1;
+  local server=$2;
+  local date_today=$(date +%s);
 
   ## Get expirationdate and convert to epoch
-  expirationdate=$(date -d "$(: | openssl s_client -connect $target:443 -servername $target 2>/dev/null \
+  expirationdate=$(date -d "$(: | openssl s_client -connect $client:443 -servername $server 2>/dev/null \
```

Comment:
these names are kinda weird, but otherwise this looks good.

Reply:
`client` & `server`?

Reply:
the `-connect ...` param is how openssl finds the _thing_ we're talking to. the `-server ...` is who it expects that thing to be, so I'd think  maybe `cloudfront` and `alias` ? or `server` and `host`?